spec/liquid_tags/dev_comment_tag_spec.rb

```diff
@@ -12,24 +12,22 @@ def generate_comment_tag(id_code)
   end
 
   context "when given valid id_code" do
-    it "fetches the target comment" do
+    it "renders properly fails because of timestamp", skip: "approvals gem does not have configuration to handle dynamically generated values" do
```

Comment:
Instead of commenting this out completely, you can write a simple assertion test that makes sure the generated html code includes the comment. Also the two approval files won't be necessary

Reply:
Just changed the tests to include the appropriate comment content and user.